package.json

```diff
@@ -9,15 +9,17 @@
   },
   "keywords": [],
   "author": "VibingDev",
-  "license": "ISC",
+  "license": "CC BY-SA 4.0",
   "dependencies": {
     "@discordjs/opus": "^0.9.0",
     "ascii-table": "^0.0.9",
     "axios": "^1.1.3",
     "discord-music-player": "^9.1.1",
     "discord.js": "^14.3.0",
     "glob": "^8.0.3",
-    "isomorphic-unfetch": "^3.1.0"
+    "isomorphic-unfetch": "^3.1.0",
+    "jimp": "^0.16.2",
+    "minecraft-protocol": "github:jtsiskin/node-minecraft-protocol"
```

Comment:
Please pin this to a specific git commit so it doesn't break when someone else downloads a newer version as this will always download the latest commit.

Reply:
Right will do